src/domains/ecommerce-app/api/ecommerce-checkout/v2/_base_policy.xml.tpl

```diff
@@ -24,8 +25,11 @@
       <value>CHECKOUT</value>
     </set-header>
       <choose>
-        <when condition="@( context.Request.Url.Path.Contains("transactions") )">
+        <when condition="@( context.Request.Url.Path.Contains("transactions") && context.Operation.Id.Equals("newTransaction")  )">
           <set-backend-service base-url="@("https://${ecommerce_ingress_hostname}"+context.Variables["blueDeploymentPrefix"]+"/pagopa-ecommerce-transactions-service/v2.1")"/>
+        </when>
```

Comment:
please add policy for GET transaction for jwt token validation (see [here](https://github.com/pagopa/pagopa-infra/blob/c4267a2478075d3bd5282a2db17caa8786398d3c/src/domains/ecommerce-app/04_apim_ecommerce_checkout.tf#L102-L109) for an example of how it have been done in v1 version)

Reply:
done here 3348cb054271cef8d7f71fde6c4c5f77ab15b03c